### PR TITLE
Error handling 

### PR DESCRIPTION
**Pull request by jasper-howard**

### CODE REVIEW

File: src/components/AddComment.jsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const AddComment = ({ topic, article_id }) => {
   const [submitted, setSubmitted] = useState(false);
   const [commentRes, setCommentRes] = useState({});
   const {
-    user: { user }, // why are you mad ?
```
Comment (Rosem10) [inline src/components/AddComment.jsx:14]: Been there, my friend.

File: src/api.js [hunk 2]
```diff
@@ -18,10 +18,10 @@ const fetchArticle = (article_id) => {
...
 
-const updateVotes = (article_id) => {
+const updateVotesArticle = (article_id, value) => {
   return axios
     .patch(`https://nc-be-news.herokuapp.com/api/articles/${article_id}`, {
-      inc_votes: 1,
+      inc_votes: value,
     })
     .catch((err) => {
       console.log(err);
```
Comment (Rosem10) [inline src/api.js]: We're not doing anything with this error? We should be able to see your error component if the vote is unsuccessful for whatever reason.

File: src/components/AddComment.jsx [hunk 3]
```diff
@@ -20,15 +20,20 @@ const AddComment = ({ topic, article_id }) => {
...
-      setCommentRes(data.comment);
-      setSubmitted(true);
-    });
+    })
+      .then(({ data }) => {
+        setCommentRes(data.comment);
+        setSubmitted(true);
+      })
+      .catch((err) => {
+        console.log(err);
```
Comment (Rosem10) [inline src/components/AddComment.jsx]: Again, rather than just console.logging the error, we should feedback to the user that there has been an error.

**Review by Rosem10 — APPROVED** [review]

File: src/components/AddComment.jsx (hunk 1 above)
Comment (jasper-howard) [reply src/components/AddComment.jsx:14]: haha forgot i left that in

File: src/api.js (hunk 2 above)
Comment (jasper-howard) [reply src/api.js]: ooop sorry forgot the comment errors will do that now

**Pull request merged**